vhost-device-template/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+name = "vhost-device-template"
+version = "0.1.0"
+authors = ["Viresh Kumar <viresh.kumar@linaro.org>"]
+description = "Template for a vhost backend device"
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["template", "vhost", "virt", "backend"]
+license = "Apache-2.0 OR BSD-3-Clause"
+edition = "2021"
```

```suggestion
edition = "2021"
publish = false
```

https://doc.rust-lang.org/cargo/reference/manifest.html#the-publish-field